Sources/Controllers/2.0/OADeleteCustomFiltersViewController.mm

```diff
@@ -181,13 +181,26 @@ - (nonnull UITableViewCell *)tableView:(nonnull UITableView *)tableView cellForR
         if (cell)
         {
             OAPOIUIFilter *filter = (OAPOIUIFilter *) item;
-            UIImage *poiIcon = [UIImage templateImageNamed:filter.getIconId];
-            cell.imgView.image = poiIcon ? poiIcon : [UIImage templateImageNamed:@"ic_custom_search_categories"];
-            cell.textView.text = filter.getName ? filter.getName : @"";
-            cell.descriptionView.hidden = true;
             BOOL selected = [_selectedItems containsObject:filter];
+
+            UIImage *icon;
+            NSObject *res = [filter getIconResource];
+            if ([res isKindOfClass:[NSString class]])
+            {
+                NSString *iconName = (NSString *)res;
+                icon = [[OAUtilities getMxIcon:iconName] imageWithRenderingMode:UIImageRenderingModeAlwaysTemplate];
+            }
+            if (!icon)
+                icon = [UIImage templateImageNamed:@"ic_custom_search_categories"];
+            [cell.imgView setImage:icon ];
             UIColor *selectedColor = selected ? UIColorFromRGB(color_primary_purple) : UIColorFromRGB(color_tint_gray);
             cell.imgView.tintColor = selectedColor;
+            cell.imgHeightPrimary.constant = 24.0;
```

Comment:
All image views for icons should be 30x30. Figma only shows the size of the actual image content and does not accound for the size of a png (which is 30x30) for our project